plugins/modules/restore.py

```diff
@@ -7,197 +7,196 @@
-        This allows for faster file parsing.
+    - Switch to indicate the backup files are in a folder structure as created by Ola Hallengreen's maintenance scripts.
+    - This allows for faster file parsing.
     type: bool
     required: false
     default: false
   ignore_log_backup:
     description:
-      - Indicates to skip restoring any log backups.
+    - Indicates to skip restoring any log backups.
     type: bool
     required: false
     default: false
   ignore_diff_backup:
     description:
-      - Indicates to skip restoring any differential backups.
+    - Indicates to skip restoring any differential backups.
     type: bool
     required: false
     default: false
   use_destination_default_directories:
     description:
-      - Switch that tells the restore to use the default Data and Log locations on the target server.
-        If they don't exist, the function will try to create them.
+    - Switch that tells the restore to use the default Data and Log locations on the target server.
+    - If they don't exist, the function will try to create them.
     type: bool
     required: false
     default: false
   reuse_source_folder_structure:
     description:
-      - By default, databases will be migrated to the destination Sql Server's default data and log directories.
-        You can override this by using C(reuse_source_folder_structure).
+    - By default, databases will be migrated to the destination Sql Server's default data and log directories.
+    - You can override this by using C(reuse_source_folder_structure).
     type: bool
     required: false
     default: false
   destination_file_prefix:
     description:
-      - This value will be prefixed to ALL restored files (log and data).
+    - This value will be prefixed to B(all) restored files (log and data).
     type: str
     required: false
   restored_database_name_prefix:
     description:
-      - A string which will be prefixed to the start of the restore Database's name.
+    - A string which will be prefixed to the start of the restore Database's name.
     type: str
     required: false
   directory_recurse:
     description:
-      - If specified the specified directory will be recursed into (overriding the default behaviour).
+    - If specified the specified directory will be recursed into (overriding the default behaviour).
     type: bool
     required: false
     default: false
   standby_directory:
     description:
-      - If a directory is specified the database(s) will be restored into a standby state,
-        with the standby file placed into this directory (which must exist, and be writable by the target Sql Server instance).
+    - If a directory is specified the database(s) will be restored into a standby state,
+    - with the standby file placed into this directory (which must exist, and be writable by the target Sql Server instance).
```

Comment:
In this case it should not be a new list item since it's the same sentence, I think.
```suggestion
    - If a directory is specified the database(s) will be restored into a standby state,
      with the standby file placed into this directory (which must exist, and be writable by the target Sql Server instance).
```